Firestore/Source/API/FIRFirestore.m

```diff
@@ -279,6 +279,16 @@ + (void)enableLogging:(BOOL)logging {
   FIRSetLoggerLevel(logging ? FIRLoggerLevelDebug : FIRLoggerLevelNotice);
 }
 
+- (void)enableNetworkWithCompletion:(void (^ _Nullable)(NSError *_Nullable error))completion {
```

Comment:
I'm pretty sure the type of the argument here can be

```
nullable void (^)(NSError *_Nullable error)
```

Possibly it can even be
```
nullable void (^)(nullable NSError *error)
```

In general we've been favoring the `nullable` attribute over the `_Nullable` qualifier.

Reply:
Ok, noted. FWIW, first option works, second does not.